readme.md

```diff
@@ -175,6 +175,7 @@
 	- [Containers](https://github.com/Friz-zy/awesome-linux-containers#readme)
 	- [eBPF](https://github.com/zoidbergwill/awesome-ebpf#readme) - Virtual machine that allows you to write more efficient and powerful tracing and monitoring for Linux systems.
 	- [Arch-based Projects](https://github.com/PandaFoss/Awesome-Arch#readme) - Linux distributions and projects based on Arch Linux.
+	- [AppImage](https://github.com/AppImage/awesome-appimage#readme) - Cross-distribution format and tools to package applications for Linux as single files.
```

Comment:
I think the description could be shorter and clearer.

Maybe this?

> Package apps in a single file that works on various mainstream Linux distributions.

Reply:
Thanks, done.